sympy/printing/tests/test_latex.py

```diff
@@ -1911,4 +1912,4 @@ def unimplemented_expr_sup_sub(expr):
 
     assert latex(unimplemented_expr(x)) == r'UnimplementedExpr\left(x\right)'
     assert latex(unimplemented_expr(x**2)) == r'UnimplementedExpr\left(x^{2}\right)'
-    assert latex(unimplemented_expr_sup_sub(x)) == r'UnimplementedExpr^{1}_{x}\left(x\right)'
+    assert latex(unimplemented_expr_sup_sub(x)) == r'UnimplementedExpr^{1}_{x}\left(x\right)'
```

Comment:
If you run `python bin/test quality` before pushing you will notice that you need a return here. Every file must end with an empty line.